app/src/components/Project/Header.tsx

```diff
@@ -3,34 +3,46 @@ import Logo from '../ui/Logo'
 import UserPanel from '../User/UserPanel'
 import { useAuth } from '../../context/AuthContext'
 import { useProject } from '../../context/ProjectContext'
+import { SearchIcon, RingIcon } from '../icons'
 
 export function ProjectHeader() {
   const { id: projectId = '' } = useParams()
   const navigate = useNavigate()
   const { user } = useAuth()
   const { search, setSearch } = useProject()
+  // const badgeCount = 5
 
   return (
-    <header className='flex justify-between items-center border-b-1 bg-grayBg h-14 border-border-color'>
+    <header className='flex justify-between items-center border-b bg-[#edebf3] h-14 border-border-color'>
       <div
         onClick={() => navigate(`/${projectId}`)}
-        className='w-80 border-r-1 border-border-color p-2 text-4xl text-center  text-black hover:cursor-pointer'
+        className='w-80 h-full border-r border-border-color p-2 flex justify-center items-center text-black hover:cursor-pointer'
       >
         <Logo />
       </div>
-      <div className='justify-self-start pl-6 flex items-center flex-1'>
-        <i className='absolute ri-search-line text-xl text-gray-400' />
+      <div className='flex-1 flex items-center px-6 gap-3 relative bg-[#edebf3]'>
+        <SearchIcon className='absolute left-0  w-5 h-5 text-[#969696]' />
         <input
-          className='block pl-7 align-middle overflow-hidden border-none bg-grayBg text-sm font-monaspace'
-          onChange={(e) => setSearch(e.target.value)}
+          type='text'
           value={search}
+          onChange={(e) => setSearch(e.target.value)}
           placeholder='Find it all'
+          className='bg-[#edebf3] text-[#969696] text-sm font-normal font-agron'
         />
       </div>
-
-      <div className='h-full flex items-center p-5 '>
-        <UserPanel user={user} />
+      <div className='flex px-7 py-2 items-center h-full w-48 border-l border-border-color  justify-center '>
+        <div className='flex items-center gap-4'>
+          <RingIcon className='w-5 h-5' />
+          <span className="text-[#46434e] text-xs font-normal font-['Monaspace Argon Var'] ml-1">
+            {/* +{badgeCount} */}
+          </span>

```

Comment:
You could commented this one and remove ```typescript
const badgeCount = 5
``` at all

Reply:
and yes, you could just use `font-monaspace` for this font, it's added to tailwind theme 🫠